README.md

```diff
@@ -1,25 +1,38 @@
-# Forks and Clones
+# Forking and Cloning Github Repositories
+
+## Problem Statement
+
+It will be important to get used to forking and cloning repos. Both are `git` functions
+that allow a user to make copies of the repo, but can serve different functions, depending
+on what needs to be done. If you're working on a repository that you don't have "write"
+access to, such as Flatiron School's `github.com/learn-co-students/` repo, you may find
+yourself cloning a readme or lab to your local environment, but being unable to submit
+your changes with a `git push`. How are you able to keep track of your changes that are
+being added to a repo you're working on? This is where you would use `git fork` and make
+a clone of that instead!
 
 ## Objectives
 
-1. Fork a repo
-2. Clone a repo
+1. Use `git fork` to copy a repo
+2. Use `git clone` to save a repo to your local machine
 
-## Forking a GitHub Repository
+## Use `git fork` to Copy a Repo
 
-Forking a GitHub repository is just a way to create your own copy of it. We do
-this all the time while using Learn. Every time you fork a lab, GitHub creates
-a copy that you can use as a sandbox to play around, all while maintaining a
-canonical repo where the original lab remains intact. In other words, forking
-creates another copy that allows you to work in your own world without messing
-with the core learn-co-students content.
+Forking a GitHub repository is just a way to create your own copy of it. For those 
```

Comment:
This intro sent. is problematic. `copy` is overloaded when applied to `fork` and `clone`. See my example text below. The verbs here are very sticky and require **absolute consistency**. It's a fault of git to use `clone` instead of `dupe`.

This sentence is really fraught and doesn't add a lot of value. Focus on making the other sentences in this para that really tell what's up shine and cut, cut, cut.